include/detectcone.hpp

```diff
@@ -17,127 +17,125 @@
  * USA.
  */
 
-#ifndef OPENDLV_LOGIC_CFSD18_SENSATION_ATTENTION_HPP
```

Comment:
I am also getting warnings/errors from cv::Mat at detectcone.cpp:580
'/opt/sources/src/detectcone.cpp: In member function 'void DetectCone::forwardDetectionORB(cv::Mat)':
/opt/sources/src/detectcone.cpp:580:38: error: array must be initialized with a brace-enclosed initializer
   cv::Mat probMap[4] = cv::Mat::zeros(m_height, m_width, CV_64F);
                        ~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~'